packages/app/src/systems/Transaction/hooks/useTransaction.tsx

```diff
@@ -23,7 +23,9 @@ const selectors = {
 };
 
 export function useTransaction(id: string) {
-  const { isLocked, isLoading, wallet } = useAccount();
+  const wallet = undefined;
+  const isLocked = true;
```

Comment:
Why have these two been set as mocked values here?

Reply:
we don't have global `isLocked` and `wallet` anymore.
will be implemented in `txApproval` like it is in `signMessage`.

> txApproval is gonna be broken until unlockMachine code changes are applied(Ref https://github.com/FuelLabs/fuels-wallet/issues/158 and https://github.com/FuelLabs/fuels-wallet/issues/159), which is gonna be my immediate next PR.